kafka-rest/src/main/java/io/confluent/kafkarest/controllers/SchemaManagerImpl.java

```diff
@@ -109,12 +124,19 @@ private RegisteredSchema getSchemaFromSchemaId(
 
   private int getSchemaVersion(String subject, ParsedSchema schema) {
     try {
-      return schemaRegistryClient.getVersion(subject, schema);
+      return schemaRegistryClient.get().getVersion(subject, schema);
     } catch (IOException | RestClientException e) {
-      throw new SerializationException(
+      String schemaString;
+      if (schema.canonicalString().equals("\"int\"")) {
```

Comment:
Can you add more detail about this? What's special about the `"int"` schema? Can you maybe write a comment about this special case?

Reply:
I think I retract this with another look at the test case that I thought caused it 😊